web/modules/custom/itk_cookie_message/itk_cookie_message.module

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * @file
+ * This module shows a message on cookies being used in the site.
+ *
+ * The EU Directive on Privacy and Electronic Communications.
+ */
+
+/**
+ * Implements hook_theme().
+ */
+function itk_cookie_message_theme($existing, $type, $theme, $path) {
+  return [
+    'itk_cookie_message_block' => [
+      'template' => 'itk-cookie-message-block',
+      'variables' => ['variables' => NULL],
+    ],
+  ];
+}
+
+/**
+ * Get settings for cookie messages.
+ *
+ * @param \Drupal\Core\Language\LanguageInterface $language
+ *   The language to use for translations. Defaults to current language.
+ *
+ * @return array
+ *   The (localized) settings.
+ */
+function _itk_cookie_message_get_settings(\Drupal\Core\Language\LanguageInterface $language = NULL) {
```

Comment:
Should languge interface here have been in a use statement;

`use \Drupal\Core\Language\LanguageInterface`